oktapam/constants/attributes/attributes.go

```diff
@@ -30,6 +30,12 @@ const (
 	CharacterOptions                  = "character_options"
 	Claims                            = "claims"
 	ClientSessionDuration             = "client_session_duration"
+	CloudConnection                   = "cloud_connection"
```

Comment:
nit : This attribute is not used anywhere

Reply:
Removed